### PR TITLE
Updated existing docs and added new information about the Teams app

### PR DESCRIPTION
**Pull request by anneguo3**
Note: Images have not been added yet - there are placeholders for the time being, but please review the updated content about the new features

### CODE REVIEW

File: concepts/graph-explorer/graph-explorer-features-teams-app.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+The response data shows up with:
+1. HTTP response code
+2. Response from Microsoft Graph API with data
+3. Response headers
+
+![Screenshot of a sample request in Graph Explorer](./images/making-a-get-request-teams-app.png)
+
+## Granted resource-specific consent
+The list of granted resource-specific consent allows you to understand what permissions have been granted to the app, and allows you to cross-reference with the permissions required for a request. To learn more about resource specific consent, view the [documentation](https://aka.ms/teams-rsc).
```
Comment (acchiang) [inline concepts/graph-explorer/graph-explorer-features-teams-app.md]: don't forget the hyphen between resource and specific, in the last sentence

File: concepts/graph-explorer/graph-explorer-overview.md [hunk 2]
```diff
@@ -9,14 +9,15 @@ author: bettirosengugi
 
 # Use Graph Explorer to try Microsoft Graph APIs
 
-[Graph Explorer](https://developer.microsoft.com/graph/graph-explorer/) is a developer tool that lets you conveniently make Microsoft Graph REST API requests and view corresponding responses. Use Graph Explorer to try APIs on the default sample tenant to explore capabilities, or sign in to your own tenant and use it as a prototyping tool to fulfill your app scenarios. This tool includes helpful features such as code snippets in C#, Java, JavaScript, and Objective C; Microsoft Graph Toolkit and adaptive cards integration; and more.
+[Graph Explorer](https://developer.microsoft.com/graph/graph-explorer/) is a developer tool that lets you conveniently make Microsoft Graph REST API requests with delegated and resource specific consent permissions and view corresponding responses. Use Graph Explorer to try APIs on the default sample tenant to explore capabilities, or sign in to your own tenant and use it as a prototyping tool to fulfill your app scenarios. This tool includes helpful features such as code snippets in C#, Java, JavaScript, and Objective C; Microsoft Graph Toolkit and adaptive cards integration; and more.
```
Comment (acchiang) [inline concepts/graph-explorer/graph-explorer-overview.md]: the hyphen between resource and specific

File: concepts/graph-explorer/graph-explorer-overview.md [hunk 3]
```diff
@@ -77,4 +78,4 @@ To view the response in a format other than the default JSON, choose the **Reque
 
 - Visit [Graph Explorer](https://developer.microsoft.com/graph/graph-explorer/).
 - Learn more about [Graph Explorer features](./graph-explorer-features.md).
-- Contribute or provide feedback in the [GitHub repo](https://github.com/microsoftgraph/microsoft-graph-explorer-v4/issues/new/choose).
+- Contribute or provide feedback in the [GitHub repository](https://github.com/microsoftgraph/microsoft-graph-explorer-v4/issues/new/choose).
```
Comment (acchiang) [inline concepts/graph-explorer/graph-explorer-overview.md:81]: 👏

**Review by acchiang — APPROVED** [review]
minor typos but lgtm

File: concepts/graph-explorer/graph-explorer-features.md [hunk 4]
```diff
@@ -10,9 +10,17 @@ author: bettirosengugi
 
 [Graph Explorer](https://developer.microsoft.com/graph/graph-explorer/) is a developer tool that lets you conveniently make Microsoft Graph REST API requests and view corresponding responses. This article describes how to use some of the important features in Graph Explorer.
 
-## Consent to permissions
+[Graph Explorer Sample Teams](TODO) exists alongside the web application to mock Teams app permissions by showing resource-specific consent and is used to support "Teams application" mode in the web application. 
 
-The user or administrator must grant Graph Explorer the correct permissions via a consent process to access data in Microsoft Graph. Consent to permissions in Graph Explorer either via the **Modify permissions** tab or the **Select permissions** option in the settings gear next to your profile when you’re signed in. 
+## Use Graph Explorer as sample Teams application
```
Comment (omersayshi) [inline concepts/graph-explorer/graph-explorer-features.md:15]: Did we decide on calling it a 'sample app'? I remember previous discussions with Darrel 'proxy' or just 'as a Teams application' makes more sense? Or is more "intuitive"

Comment (anneguo3) [reply concepts/graph-explorer/graph-explorer-features.md:15]: The name needs to be reviewed with Laura - Ayu has it on her list of things to cover with her

**Review by lhao03 — APPROVED** [review]
noice 😊
(～￣▽￣)～

**Pull request merged**